docs/getting-started.md

```diff
@@ -0,0 +1,140 @@
+﻿
+# Kubevirtci
+
+Kubevirtci rule is to create a pre provisioned clusters, publish them as a container image,\
```

Comment:
kubevirtci's role is to create preprovisioned Kubernetes/OpenShift clusters as container images.